### ISSUE TITLE
[resolved] OSt 1.14 RC2 Copy past text in reply field

### ISSUE THREAD

**Issue by ghost** [osTicket/osTicket#5161]
OSt 1.14 RC2

I copy  some (rich) text into the reply field. This causes the text into a narrow column.  Try it using the lorem generator: https://nl.lipsum.com/feed/html

Copy and past some text into the reply field and see what happens. <this happens also by copy/past from a rich text editor into the the reply field. 

![Schermafdruk van 2019-11-12 11-32-48](https://user-images.githubusercontent.com/49860071/68664189-37c62600-0540-11ea-8355-46e6fc6df540.png)

**Comment by JediKev** [comment]
@bernardflach 

I cannot replicate the same issue. I went to the site you linked and copied the first two paragraphs:
<img width="935" alt="Screen Shot 2019-11-12 at 09 04 37" src="https://user-images.githubusercontent.com/11823401/68682946-9339e900-052b-11ea-879b-b20750999501.png">

After that I pasted into the Text Editor and it was just fine:
<img width="929" alt="Screen Shot 2019-11-12 at 09 04 30" src="https://user-images.githubusercontent.com/11823401/68682990-a3ea5f00-052b-11ea-910b-01c74e44b3d6.png">

Maybe you have something weird going on with your clipboard?

Cheers.
